pep-0707.rst

```diff
@@ -2,14 +2,13 @@ PEP: 707
 Title: A simplified signature for __exit__ and __aexit__
 Author: Irit Katriel <iritkatriel@gmail.com>
 Discussions-To: https://discuss.python.org/t/24402
-Status: Draft
+Status: Rejected
 Type: Standards Track
 Content-Type: text/x-rst
 Created: 18-Feb-2023
 Python-Version: 3.12
 Post-History: `02-Mar-2023 <https://discuss.python.org/t/24402/>`__,
-Resolution:
-
+Resolution: `10-Apr-2023 <https://github.com/python/steering-council/issues/178#issuecomment-1502209587>`__.
```

Comment:
Sorry if it was confusing, but this should point to the post in the Discussions-To thread officially rejecting the PEP (analogous to the old post to `python-dev`) rejecting the PEP, rather than the SC issue tracker. I.e.

```suggestion
Resolution: https://discuss.python.org/t/pep-707-a-simplified-signature-for-exit-and-aexit/24402/46
```

It would be great to include the date in the resolution and something I've wanted to implement but a bit, I just haven't gotten to it.

Thanks!